src/e2eIosTest/java/io/appium/java_client/ios/BaseIOSWebViewTest.java

```diff
@@ -27,17 +27,19 @@
 public class BaseIOSWebViewTest extends BaseIOSTest {
     private static final String VODQA_ZIP = TestUtils.resourcePathToAbsolutePath("vodqa.zip").toString();
 
-    private static final Duration WEB_VIEW_DETECT_INTERVAL = Duration.ofSeconds(1);
-    private static final Duration WEB_VIEW_DETECT_DURATION = Duration.ofSeconds(15);
+    private static final Duration WEB_VIEW_DETECT_INTERVAL = Duration.ofSeconds(2);
+    private static final Duration WEB_VIEW_DETECT_DURATION = Duration.ofSeconds(30);
 
     @BeforeAll
     public static void beforeClass() {
         startAppiumServer();
 
         XCUITestOptions options = new XCUITestOptions()
+                .setPlatformVersion(PLATFORM_VERSION)
                 .setDeviceName(DEVICE_NAME)
                 .setWdaLaunchTimeout(WDA_LAUNCH_TIMEOUT)
                 .setCommandTimeouts(Duration.ofSeconds(240))
+                .setShowIosLog(true)
```

Comment:
I would keep it for some time to monitor the tests, there is some flakiness in the app, it will help to catch details